terraform/ec2/README.md

```diff
@@ -275,7 +275,16 @@ Outputs:
 ### Notes
 [reference of how to create role](https://github.com/aws-actions/configure-aws-credentials)
 
-## 7. Run the integration test action on your fork
+## 7. Setup the integration test resources
+1. Navigate to your fork
+2. Run the following command to setup all the resources being used for integration test (e.g vpc, iam, etc)
+```shell
+cd terraform/setup # assuming you are still in the ./terraform
+terraform init
+terraform apply --auto-approve 
+```
```

Comment:
Its a one-time setup for resources can be share (e.g vpc) and can be used all over again without determine/affect the test's outcome or make the test more deterministics. Therefore, we don't need to destroy the resources and used it over again across the workflow

Reply:
The `common` folder is a shared common constant between [these terraform](https://github.com/aws/amazon-cloudwatch-agent-test/blob/6f8c6538f088d6d3b7b8e0b0fcbe83f7f195a40f/terraform/common/output.tf). Example would be if you create an EC2 Instance Profile with name `cwa-e2e-iam-instance-profile`; therefore, it will look up them with the same name (e.g [EC2 Terraform looking up](https://github.com/aws/amazon-cloudwatch-agent-test/blob/6f8c6538f088d6d3b7b8e0b0fcbe83f7f195a40f/terraform/ec2/linux/iam.tf#L4-L6))